dockers/docker-lldp/lldpmgrd

```diff
@@ -148,6 +177,36 @@ class LldpManager(daemon_base.DaemonBase):
         for port_name in to_delete:
             self.pending_cmds.pop(port_name, None)
 
+    def lldp_check_mgmt_info_change(self):
+        (status, fvp) = self.device_table.get("localhost")
+        if status:
+            # Convert list of tuples to a dictionary
+            device_table_dict = dict(fvp)
+
+            hostname = device_table_dict.get("hostname")
+            if not self.hostname == hostname:
+                self.log_info("Hostname changed old {0}, new {1}".format(self.hostname, hostname))
+                self.hostname = hostname
+                self.update_hostname(hostname)
+
+        mgmt_intf_keys = self.mgmt_table.getKeys()        
+        if not mgmt_intf_keys:
+            if not self.mgmt_ip == "None":
+                self.update_mgmt_addr("None")
+                self.mgmt_ip = "None"
+        else:
+            for key in mgmt_intf_keys:
+                if '|' in key:
+                    key = key.split('|', 1)
+                    if '/' in key[1]:
+                        ip = key[1].split('/', 1)
+                        if not self.mgmt_ip == ip[0]:
+                            self.log_info("Mgmt IP changed old ip {0}, new ip {1}".format(self.mgmt_ip, ip[0]))
+                            self.mgmt_ip = ip[0]
```

Comment:
suggest to move this line after making sure command is successful

Reply:
Will make the change